src/Npgsql/Internal/Postgres/DataTypeNames.cs

```diff
@@ -51,6 +51,7 @@ static class DataTypeNames
     public static DataTypeName TimeTz => ValidatedName("pg_catalog.timetz");
     public static DataTypeName Inet => ValidatedName("pg_catalog.inet");
     public static DataTypeName Cidr => ValidatedName("pg_catalog.cidr");
+    public static DataTypeName Cube => ValidatedName("pg_catalog.cube");
```

Comment:
The `cube` extension type can be installed in any schema not just `pg_catalog`, typically it is installed in `public`, but the user may choose to install it somewhere else. Therefore, this qualifier is actually invalid.